joomla/joomla-cms
# Debuguser and debuggroup legend

#### Summary of Changes

The legend just looked wrong and it was because there was a space between the icon and the text which made it look like the wrong icon was associated with the text
#### Testing Instructions

If you enable debug in global configuration and then visit the users and groups pages there is a debug option. When selected you will see a legend at the top
#### Before

![xwa9](https://cloud.githubusercontent.com/assets/1296369/14742867/486f8be0-0896-11e6-9b32-8b16731a400d.png)
#### After

![v3ox](https://cloud.githubusercontent.com/assets/1296369/14742888/6186bb30-0896-11e6-8d7c-c109ed49921f.png)
